### PR TITLE
UCT/TEST: Fix testing of self transport

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/5833/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/5837/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/5840/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
@hoopoepg @brminich pls take a look

File: src/ucp/core/ucp_context.c [hunk 1]
```diff
@@ -334,35 +354,25 @@ static int ucp_is_resource_enabled(const uct_tl_resource_desc_t *resource,
...
-                    ucs_trace("enabling auxiliary tl '%s' for alias '%s'",
-                              resource->tl_name, alias->alias);
-                    break;
-                }
+            count = ucp_tl_alias_count(alias);
+            snprintf(info, sizeof(info), "for alias '%s'", alias->alias);
+            tmp_flags = 0;
+            if (ucp_config_is_tl_enabled(config, alias->alias, 1, &tmp_flags) &&
+                ucp_tls_array_is_present(alias->tls, count, resource->tl_name,
+                                     info, &tmp_flags)) {
```
Comment (brminich) [inline src/ucp/core/ucp_context.c]: indentation

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/5846/ for details (Mellanox internal link).

**Pull request merged**